clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -52,6 +65,127 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Gets a load balancer pool for a given region and load balancer UUID.
+   * @param region
+   * @param loadBalancerId
+   * @return
+   */
+  LbPool getLoadBalancerPool(final Region region, final String loadBalancerId) {
```

Comment:
I figured what you have for regions is good in the PR and we will follow up and add regions to the other operations after the `getRegionClient` method is in.